meta/TreeMasses.m

```diff
@@ -408,6 +409,20 @@
 
 IsElectricallyCharged[par_] := GetElectricCharge[par] != 0;
 
+IsChargedUnder[field_, vector_] := Block[{},
```

Comment:
I'm not sure if this function should be here or should be local to FFV module. I don't think there's need for it outside of FFV calculation.

Reply:
It is done.  Thanks!